invenio_indexer/ext.py

```diff
@@ -52,6 +53,13 @@ def init_app(self, app):
         self.init_config(app)
         app.extensions['invenio-indexer'] = self
 
+        hooks = app.config.get('INDEXER_BEFORE_INDEX_HOOKS', [])
+        for hook in hooks:
+            if callable(hook):
+                before_record_index.connect_via(app)(hook)
+            elif isinstance(hook, str):
```

Comment:
@zazasa please make sure that it's both PY2/3 friendly. See https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/utils.py#L76-L87